change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -1854,12 +1856,20 @@ PROCEDURE VALIDATECONFIG
      PARAMVALIDATIONERROR=TRUE
```

Comment:
I think this line of code at Line 1649 Above this:
 "BNODEBUGMSG="ES:"+PARAMEMAILFROM"
 Should be
 "BNODEBUGMSG="ES:"+PARAMEMAILSUBJECT"

Reply:
Yes, you are right! I updated that and found another issue while I was in there... Thanks!